src/v/ssx/task_local_ptr.h

```diff
@@ -50,7 +51,12 @@ class task_local {
       , _task(get_current_task()) {}
 
     explicit task_local(T&& val)
-      : _val(val)
+      : _val(std::move(val))
+      , _task(get_current_task()) {}
+
+    template<class... Args>
+    explicit task_local(std::in_place_t, Args... args)
+      : _val(std::in_place, args...)
```

Comment:
should be Args&&... args and std::forward<Args>(args)...

Reply:
fixed